### PR TITLE
Authorization cleaning

### PR DESCRIPTION
**Pull request by lmazuel**

### CODE REVIEW

**Comment by lmazuel** [comment]
@AutorestCI regenerate azure-sdk-for-python

File: specification/authorization/resource-manager/readme.md [hunk 1]
```diff
@@ -45,19 +45,13 @@ These settings apply only when `--tag=package-2015-07` is specified on the comma
 
 ``` yaml $(tag) == 'package-2015-07'
 input-file:
-- Microsoft.Authorization/stable/2015-07-01/authorization.json
+- Microsoft.Authorization/stable/2015-07-01/authorization-RoleDefinitionsCalls.json
+- Microsoft.Authorization/stable/2015-07-01/authorization-ProviderOperationsCall.json
```
Comment (amarzavery) [inline specification/authorization/resource-manager/readme.md]: nit: might as well end the filename in *Calls.json like other files, to be consistent.

Comment (lmazuel) [reply specification/authorization/resource-manager/readme.md]: done!

File: specification/authorization/resource-manager/readme.md [hunk 2]
```diff
@@ -45,19 +45,13 @@ These settings apply only when `--tag=package-2015-07` is specified on the comma
 
 ``` yaml $(tag) == 'package-2015-07'
```
[the remark this answers is not in the document]
Comment (lmazuel) [reply specification/authorization/resource-manager/readme.md]: No, I'm fine doing it here, let me add a commit. Thanks!

[the remark this answers is not in the document]
Comment (lmazuel) [reply specification/authorization/resource-manager/readme.md]: pushed!

**Review by rlrossiter — APPROVED** [review]

**Pull request merged**